chgnet/trainer/trainer.py

```diff
@@ -285,7 +285,7 @@ def train(
             raise ValueError("Model needs to be initialized")
         global best_checkpoint  # noqa: PLW0603
         if save_dir is None:
-            save_dir = f"{datetime.now():%m-%d-%Y}"
+            save_dir = f"{datetime.datetime.now(tz=datetime.timezone.utc):%m-%d-%Y}"
```

Comment:
maybe better to use the user's own timezone here? not sure? would be
```suggestion
            save_dir = f"{datetime.datetime.now().astimezone():%m-%d-%Y}"
```

Reply:
good points. using UTC means you often have to do the mental math from your timezone to UTC to understand how long ago a file/dir was actually written but that's probably the lesser of two evils